--- netcdf-4.6.0/libdispatch/ddispatch.c.orig	2018-03-06 08:50:20.158294200 +0100
+++ netcdf-4.6.0/libdispatch/ddispatch.c	2018-03-06 08:52:13.072722200 +0100
@@ -80,8 +80,12 @@
 #ifdef _MSC_VER
         tempdir = getenv("TEMP");
 #else
+#ifdef _WIN32
+    tempdir = getenv("TEMP");
+#else
 	tempdir = "/tmp";
 #endif
+#endif
         if(tempdir == NULL) {
 	    fprintf(stderr,"Cannot find a temp dir; using ./\n");
 	    tempdir = getcwd(cwd,sizeof(cwd));
